backend/habitat/viewsets.py

```diff
@@ -93,6 +93,26 @@
 SQL_GET_ZONES = "select name from SeamapAus_Zones_View;"
 SQL_GET_ZONES_IUCN = "select name from SeamapAus_ZonesIUCN_View;"
 
+SQL_GET_NETWORK_AREA = "select geom.STArea() from SeamapAus_Networks_View where name=%s;"
+SQL_GET_NETWORK_HABITAT_STATS = """
+select habitat, area / 1000000 as area, 100 * area / %s as percentage
+from SeamapAus_Habitat_By_Region
+where
+  region = %s and
+  boundary_layer_id = 260 and
+  habitat_layer_id = 95;
```

Comment:
These are obviously the bits that jump out: it's _probably_ ok, at least for now, but let's keep it in mind while we look for a better way.

Reply:
Actually, the habitat layer will be fixed I think (let's clarify this with Emma ASAP), so is that part of the view even needed?  (Similarly, for the boundary layer we can probably get that id from the user's selection)

(If this is WIP you may already have this in mind)

Reply:
100% WIP. This query borrows a lot from `SQL_GET_STATS` above it. I plan to have fixed views that would eliminate that part of the query, I just haven't gotten to it because I'm still planning out the zones stuff.

Reply:
In fact, I might take a step back once I look at zones to see how I can include the necessary information in the four views I made yesterday.